energyCalibrator.py

```diff
@@ -138,7 +138,7 @@ def saturationFactorNLO(self,density):
         return ret
     
     def calibratedEnergy(self,hits):
-        slices,centers = self.getSlices(hits)
+        slices = self.getSlices(hits)
```

Comment:
@kmieckara I think that you started from an older version of the code, where the centers and integrals along the track trajectory were not saved. They are useful to make some head-tail study. Can you try to merge the latest master branch and update the pull request?